src/lib/api.js

```diff
@@ -412,12 +412,13 @@ const buildPaginationLinks = function (limit, parameters, bbox, intersects, endp
         // const query = encodeURIComponent(dict[p])
           let value = dict[p]
           if (typeof value === 'object' && value !== null) {
-            value = JSON.stringify(value)
+            if (p === 'sortby') {
+              value = value[0]['field']
```

Comment:
could you explain why the first array element and the field attribute are the right values here?